src/NHibernate/Mapping/ByCode/IClassMapper.cs

```diff
@@ -58,6 +58,7 @@ public interface IClassAttributesMapper<TEntity> : IEntityAttributesMapper, IEnt
 		void Discriminator(Action<IDiscriminatorMapper> discriminatorMapping);
 		void DiscriminatorValue(object value);
 		void Table(string tableName);
+		void Check(string check);
```

Comment:
This technically changes the interface and would be a binary breaking change. Could you please instead create an extension method for it? Also, please add some tests.